storybook/src/components/Figure/Figure.stories.tsx

```diff
@@ -15,7 +15,13 @@ const meta = {
   component: Figure,
   args: {
     children: caption,
-    inverseColor: false,
+    color: {
+      control: {
+        labels: { undefined: 'default' },
+        type: 'radio',
+      },
+      options: [undefined, 'inverse'],
+    },
```

Comment:
@VincentSmedinga I'm having some difficulty with the color prop, typescript does not seem to help getting the FigureCaptionProps.